pkg/didmethod/peer/did_test.go

```diff
@@ -81,5 +131,24 @@ func GenerateGenesisDoc() (*did.Doc, error) {
 		Created:        &time.Time{},
 	}
 	return &did.Doc{Context: doc.Context, PublicKey: doc.PublicKey, Authentication: doc.Authentication,
-		Created: doc.Created}, nil
+		Created: doc.Created}
+}
+
+func generatePeerDoc() (*did.Doc, error){
```

Comment:
Suggest we remove the "generate" prefix from all these functions.

Reply:
removed